config/config.go

```diff
@@ -1188,24 +1188,24 @@ type InstrumentationConfig struct {
 	// Instrumentation namespace.
 	Namespace string `mapstructure:"namespace"`
 
-	// InfluxURL is the influxdb url.
-	InfluxURL string `mapstructure:"influx_url"`
+	// TracePushConfig is the relative path of the push config. This second
+	// config contains credentials for where and how often to.
+	TracePushConfig string `mapstructure:"trace_push_config"`
 
-	// InfluxToken is the influxdb token.
-	InfluxToken string `mapstructure:"influx_token"`
+	// TracePullAddress is the address that the trace server will listen on for
+	// pulling data.
+	TracePullAddress string `mapstructure:"trace_pull_address"`
 
-	// InfluxOrg is the influxdb organization.
-	InfluxOrg string `mapstructure:"influx_org"`
+	// TraceType is the type of tracer used. Options are "local" and "noop".
+	TraceType string `mapstructure:"trace_type"`
```

Comment:
[question] why not continue supporting the existing config + pushing to InfluxDB and add the local tracer as a strictly new feature + new config? I think that could make this PR non-breaking.

Reply:
we have to have a fix for #1141 , which is already breaking, but also keeping the old tracer was pretty burdensome as we had to have a ton of consts and enforce the schema in a really weird way. Here, we're just using a struct, but if we keep the influx, we have to keep two separate mechanisms for enforcing the schema